central/sensor/service/service_impl.go

```diff
@@ -93,6 +100,17 @@ func (s *serviceImpl) Communicate(server central.SensorService_CommunicateServer
 	if err != nil {
 		return err
 	}
+	clusterID := cluster.GetId()
+
+	// Disallow secured cluster impersonation using a leaked CRS certificate.
+	// Note: New clusters which have never been properly connected to central have their HealthStatus.LastContact empty.
```

Comment:
Can you please add a comment in the place where `LastContact` is declared to mention that this is also used to guard against this attack vector?

Reply:
See a4216b7031c5d155c89d082370aa9d71d593e4cd